code/dqc/schemacheck.q

```diff
@@ -5,7 +5,9 @@ schemacheck:{[tab;colname;types;forkeys;attribute]
   .lg.o[`dqc;"checking schema of table mathces expectation"];
   origschema:0!meta tab;
   checkschema:([]c:colname;t:types;f:forkeys;a:attribute);
-  $[all c:((flip origschema)each key flip origschema)~'(flip checkschema)each key flip checkschema;
+  / Function to replace empty symbols with underscores in error messages.
+  f: {@[x;where 0=count each string x;{`$"_"}]};
```

Comment:
Would it make more sense to use `.Q.s1` to add backticks in front of column names?
```
q).Q.s1`a``b
"`a``b"
q).Q.s1'[`a``b]
"`a"
,"`"
"`b"
```

Reply:
I'm not sure, the problem was replacing empty symbols with underscores so that we could show where there was no attribute on a column

Reply:
Okay then I think having backticks for all is clearer than underscores.